lib/xboxrt/startup.c

```diff
@@ -3,6 +3,10 @@
 
 extern void main(void);
 
+void _chkstk() {
+    //FIXME
+}
```

Comment:
FIXME: check signature and figure out which lib we have to link or how to use a builtin

Reply:
Can be removed - fixed upstream.